sub/dec_sub.h

```diff
@@ -11,9 +12,36 @@ struct mpv_global;
 struct demux_packet;
 struct mp_recorder_sink;
 
-struct dec_sub;
 struct sd;
 
+struct dec_sub {
```

Comment:
Exposing the entire internals just for access to one option doesn't seem like a good idea to me.
I suggest adding a `bool sub_is_secondary_visible(struct dec_sub *sub);`